internal/testplanet/planet.go

```diff
@@ -158,11 +159,17 @@ func NewWithLogger(log *zap.Logger, satelliteCount, storageNodeCount, uplinkCoun
 			}))
 
 		go func(n *Node) {
```

Comment:
I'd like to see this whole go routine and timer moved into Discovery for reusability, ala kad's StartRefresh()

Reply:
Moved. 👍